src/test/rpc/GatewayBalances_test.cpp

```diff
@@ -153,6 +155,7 @@ class GatewayBalances_test : public beast::unit_test::suite
     run() override
     {
         testGWB();
+        testGWB(featureFlow, featureToStrandV2);
```

Comment:
This one doesn't need `test_bookdir(featureFlow)`?  If not it might be a good idea to include a comment why.